README.md

```diff
@@ -88,3 +88,69 @@ The levels are as follows:
 
 ![Data Alignment Analysis]("./../docs/analysis-bnb.png)
 
+
+### Filtering
```

Comment:
This comes from the other PR I think? I would remove it here since it's not really related to the other changes.

Reply:
Oh lord I like how I did the pull request equivalent of dropping glass on the floor